test/tests/shouldAdd.test.ts

```diff
@@ -150,10 +150,10 @@ it('Should support dynamic references via refPath', async () => {
 
   // I should now have two "inventory" items, with different embedded reference documents.
   const items = await Inventory.find({}).populate('kind').exec();
-  expect((items[0].kind as typeof Beverage).isDecaf).toEqual(true);
+  expect((items[0].kind as BeverageClass).isDecaf).toEqual(true);
```

Comment:
do you know why the `typeof` got unnecessary?

Reply:
Because here we expected `kind` is an instance of Beverage class, not a class itself.

Btw, CI failed, but i didn't know why.

Reply:
@googlicius i know that the ci failed, but it is an coverage reporting problem, not an problem about this pr

Reply:
More precisely, `kind` is a Beverage document, but instance of Beverage still ok.